src/Components/LandingPage/LandingPage.tsx

```diff
@@ -66,15 +68,13 @@ export const LandingPage = () => {
     <section className="pf-l-page__main-section pf-c-page__main-section">
       <ExpandableSection
         className="pf-m-light pf-u-mb-xl expand-section"
-        toggleText="Help get started with new features"
+        toggleText={`Help get started with ${
+          isBeta() ? 'Preview' : 'new'
```

Comment:
Any reason to capitalise preview?

Reply:
The only reason is that it was capitalised in the previous versions. Should I switch it to lowercase?

![Screenshot from 2023-08-23 13-04-55](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/9707b6ee-123d-467f-bb20-e95f6b158715)